### ISSUE TITLE
Investigate potential extensions to evidence codes

### ISSUE THREAD

**Issue by dosumis** [geneontology/synapse#2]
At the Broad meeting, there was extensive discussion of the need for exended evidence codes to distinguish types of experimental evidence for synaptic localisation and function.  We need to:

1. Assess the need / use-cases for extended evidence codes
1. Look into how ECO might be extended so that any new evidence codes could map up to the generally used ones.
1. Talk to Tony S about whether Protein2GO could be extended to allow for extended evidence codes.

**Comment by cmungall** [comment]
Any notes on the requirements for new evidence types?

**Comment by Pimmelorus** [comment]
Hi all,

I have been collecting evidence types while collecting evidence for synaptic proteins. Could you point me to the current set of evidence types, so that I get an idea of what is required?

All the best,

Pim

--
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-

   Pim van Nierop

   Center for Neurogenomics and Cognitive Research
   Faculty of Earth and Life Sciences
   VU University, Amsterdam
   Email    pim.van.nierop@cncr.vu.nl
   Tel.     +31 (0)20 5987114
   Fax.     +31 (0)20 5987112

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-

________________________________

**Comment by dosumis** [comment]
￼￼￼
![image](https://cloud.githubusercontent.com/assets/112839/7840605/1922ebd0-0490-11e5-8ad9-1fd21046eb1a.png)

**Comment by dosumis** [comment]
Some partial/draft mappings based on table above:

* [microscopy]([http://www.ebi.ac.uk/ontology-lookup/browse.do?ontName=ECO&termId=ECO%3A0001098&termName=microscopy%20evidence):  F, C, C' 
* [immunolocalization evidence](http://www.ebi.ac.uk/ontology-lookup/?termId=ECO%3A0000087): C, F
* [cell fractionation evidence](http://www.ebi.ac.uk/ontology-lookup/?termId=ECO%3A0000004): G, H, D
* [inferred from mutant phenotype](http://www.ebi.ac.uk/ontology-lookup/?termId=ECO%3A0000315): M, A

**Comment by dosumis** [comment]
Hi Helen (@helenp), here's the ticket on discussion of extended evidence codes I mentioned.
Cheers,
David

**Comment by dosumis** [comment]
@Pimmelorus @ftwkoopmans action item from last meeting: please could you post full tables of evidence types, either in the ticket or as tsv files on this site.  Add all the warnings you think necessary to make it clear that these are draft versions and not to be used by anyone outside of this project.

**Comment by Pimmelorus** [comment]
David, I have added the [draft for the types of experimental evidences](https://github.com/geneontology/synapse/blob/master/evidence_framework/CNCR_evidence_framework_DRAFT.ods) for the annotation to the git repository. I did not add it as a .tsv since it would destroy the formatting of the spreadsheet. Moreover, this draft should not be treated as an official document!!!! It is still my personal idea of evidence types and has not yet been internally discussed with other scientists in my institute. It has only be added to give a idea of experimental evidence.

**Comment by dosumis** [comment]
Thanks Pim.  Looks like there are some in the table above that are not in the spreadsheet.  Worth combining?

(BTW - If you're interested in Drosophila Brain regions - I'd recommend VFB/FlyBase as a source.  NeuroLex got them from us and is rather out of date).

**Comment by bmeldal** [comment]
Hi,
Sorry to butt in. I might be worthwhile looking at the PSI-MI ontology for [interaction detection methods] (http://www.ebi.ac.uk/ontology-lookup/browse.do?ontName=MI&termId=MI:0045&termName=experimental%20interaction%20detection) and [participant detection methods] (http://www.ebi.ac.uk/ontology-lookup/browse.do?ontName=MI&termId=MI:0661&termName=experimental%20participant%20identification) to see if ECO can align. There was a GO annotation call a few months ago when I suggested we might be able to help but I don't know if the 'synapsis group' were represented on the call.
PS: I'm working through the acetylecholine receptors right now and drawing up new term proposals. It'll take a little while until I have it all in place but will link the ticket to here in case anyone wants to comment. @dosumis, do you want that ticket on SF or GH (when it's ready)?
Birgit
IntAct & Complex Portal

**Comment by Pimmelorus** [comment]
@dosumis 
>Thanks Pim. Looks like there are some in the table above that are not in the spreadsheet. Worth combining?
Please do not look at this at that level! When discussing whether to join two separate documents you are already trying to drive towards a final body of evidence codes based on the evidences in the documents. The shared documents have **NO** relationship and do **NOT** represent a official proposal for evidence types. They are merely communicated to give you an idea of experimental evidences that we look at.

>BTW - If you're interested in Drosophila Brain regions - I'd recommend VFB/FlyBase as a source. NeuroLex got them from us and is rather out of date
I just included the NeuroLex reference as an example. I am open to any alternative. However, most annotations will relate to human, rat, and mouse species.

**Comment by Pimmelorus** [comment]
@bmeldal 
I think that indeed part of our evidences would relate to interaction data. However, we are also heavily focussed on localization and functional evidence.

**Comment by bmeldal** [comment]
@Pimmelorus 
I know your remit would be wider, just pointing out that on the interaction side there is a well established community standard that may make your life easier when putting the project together :)

**Comment by Pimmelorus** [comment]
@bmeldal 
Ah ok, I get it. Thnx for contributing :)

**Comment by dosumis** [comment]
@tonysawfordebi confirms that protein2GO will support use any code from eco at some point in the near future (hopefully by this autumn).  This is dependent on the release of QuickGO2 - which will provide the lookups for eco ids.  The GOA database, where this data will be stored and maintained after annotation, already supports ECO.  We may have some issues with releasing as GAF (requires mapping up to standard GO evidence codes), but can published as GPAD.

**Comment by dosumis** [comment]
@bmelda 

Good point about mapping to PSI-MI.  We may at least be able to do something quick with xrefs in ECO.  The bigger mapping and integration issues should be brought up with ECO and PSI-MI folks.

**Comment by bmeldal** [comment]
I offered our help to Marcus Chibucos when he presented ECO at an annotation TC earlier this year. I think they were struggling for developer time - as everyone else! - to do the actual integration work. Just shout when you are ready :)
Good to know all ECO codes will be available to use in the next version of QuickGO as we'll need more for the Complex annotations as well.
